torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1836,6 +1898,16 @@ std::tuple<at::Tensor&, at::Tensor&> AtenXlaType::max_out(
   return std::forward_as_tuple(max, max_values);
 }
 
+at::Tensor AtenXlaType::max_pool2d(const at::Tensor& self,
```

Comment:
Can we open an issue, maybe linked with the pytorch issue, to get rid of this one the max_pool* restructuring will happen in pytorch?